sql/wsrep_trans_observer.h

```diff
@@ -470,17 +486,19 @@ static inline
 int wsrep_after_statement(THD* thd)
 {
   DBUG_ENTER("wsrep_after_statement");
-  WSREP_DEBUG("wsrep_after_statement for %lu client_state %s "
-              " client_mode %s trans_state %s",
-              thd_get_thread_id(thd),
-              wsrep::to_c_string(thd->wsrep_cs().state()),
-              wsrep::to_c_string(thd->wsrep_cs().mode()),
-              wsrep::to_c_string(thd->wsrep_cs().transaction().state()));
   int ret= ((thd->wsrep_cs().state() != wsrep::client_state::s_none &&
                thd->wsrep_cs().mode() == Wsrep_client_state::m_local) &&
               !thd->internal_transaction() ?
               thd->wsrep_cs().after_statement() : 0);
 
+  WSREP_DEBUG("wsrep_after_statement for %lu client_state %s "
```

Comment:
As above: Relevance vs. chance of merge conflicts.

Reply:
Removed.